src/js/components/Beacon/BeaconQueryUi.tsx

```diff
@@ -113,7 +141,46 @@ const BeaconQueryUi = () => {
 
   // form utils
 
+  const variantsFormValid = (formValues: FormValues) => {
+    // valid variant form states:
+    // empty except possibly autofilled assemblyID (no variant query)
+    // chrom, start, assemblyID, end (range query)
+    // chrom, start, assemblyID, alt, ref (sequence query)
+    // https://docs.genomebeacons.org/variant-queries/
+
+    // as an alternative, we could require "end" always, then form logic would be less convoluted
+    // just set start=end to search for SNPs
```

Comment:
when we eventually have chrom autofill from gohan/ref service, we could use the information in the ref service to auto-fill `end` to the end of the chromosome if it's unset (and change it if the user changes the chromosome to the length of the new chromosome if it === the previous chrom length) or something like this. for now, i think this is fine even though it's convoluted.

Reply:
Do people really want to search entire chromosomes?

Reply:
not really no, that's true

Reply:
could perhaps work for genes though I guess